core/trino-main/src/test/java/io/trino/sql/gen/BenchmarkInCodeGenerator.java

```diff
@@ -209,6 +210,8 @@ public static void main(String[] args)
         Options options = new OptionsBuilder()
                 .verbosity(VerboseMode.NORMAL)
                 .include(".*" + BenchmarkInCodeGenerator.class.getSimpleName() + ".*")
+                .resultFormat(ResultFormatType.JSON)
+                .result(System.getProperty("java.io.tmpdir") + "/BenchmarkInCodeGenerator-result" + System.currentTimeMillis() + ".json")
```

Comment:
This should be standardized across benchmarks, so we could have a utility function for this _and_ make the file find-able.

Also, can you replace currentTimeMillis with formatted time, to more easily distinguish between today's runs and yesterday's?